provisioner/terraform/executor.go

```diff
@@ -23,27 +23,15 @@ import (
 	"github.com/coder/coder/provisionersdk/proto"
 )
 
-// initMut is a global mutex that protects the Terraform cache directory from
-// concurrent usage by path. Only `terraform init` commands are guarded by this
-// mutex.
-//
-// When cache path is set, we must protect against multiple calls to
-// `terraform init`.
-//
-// From the Terraform documentation:
-//
-//	Note: The plugin cache directory is not guaranteed to be concurrency
-//	safe. The provider installer's behavior in environments with multiple
-//	terraform init calls is undefined.
-var initMut = &sync.Mutex{}
-
 type executor struct {
+	mut        *sync.Mutex
```

Comment:
Should this still be called `initMut`, just so we're clear what it's protecting?

Reply:
it's now protecting all terraform calls that read or alter state

Reply:
so what should we call it?

Reply:
nvm